docs/README.md

```diff
@@ -1,24 +1,41 @@
-# Docs
+# Documentation
 
-This directory contains all OpenFeature Operator documentation, see table of contents below:
+This directory contain OpenFeature Operator documentation. 
 
-## Usage
+A quick start ? Follow the quick start to get OpenFeature Operator up and running in your cluster within 5 minutes.
 
-Follow the documentation below to deploy the open feature operator to your local cluster, followed by a simple example app using `curl` to evaluate a static flag.
+- [Quick Start](./quick_start.md)
+
+Follow the detailed installation guide to deploy open feature operator to your local cluster, 
 
 - [Installation](./installation.md)
-- [Getting Started](./getting_started.md)
 
 ## Configuration
 
-Configuration of the deployed sidecars is handled through the `FeatureFlagConfiguration` CRs defined in the `openfeature.dev/featureflagconfiguration` annotation of a deployed `PodSpec`. 
-> Further configuration of the operator will be possible in the future, to help contribute [click here](https://github.com/open-feature/open-feature-operator/issues)
+Configuration of the deployed sidecars is handled through the `FeatureFlagConfiguration` custom resources defined in the `openfeature.dev/featureflagconfiguration` annotation of a deployed `PodSpec`.
```

Comment:
I think this isn't really the best explanation.

`FeatureFlagConfiguration` actually is inteded most to define flags, while `FlagSourceConfiguration` controls the sidecars.

Reply:
Thank you 🙌 

A good catch and I think naming is a little misleading and confusing sometimes. 

I updated the documentation to correct this